updater/lib/dependabot/updater/operations/update_all_versions.rb

```diff
@@ -8,12 +8,8 @@ module Dependabot
   class Updater
     module Operations
       class UpdateAllVersions
-        def self.applies_to?(job:)
-          return false if job.security_updates_only?
-          return false if job.updating_a_pull_request?
-          return false if job.dependencies&.any?
-
-          true
+        def self.applies_to?(_job:)
+          false # only called elsewhere
```

Comment:
was the previous applies_to? implementation here a mistake? Or was this being called before, and with your changes it's not?

Reply:
oh I think I understand. 

`UpdateAllVersions` was being called directly from `GroupUpdateAllVersion#directories`, so we needed the applies_to? method to still run the right checks, but now that every multi-dir and grouped PR is going to use `GroupUpdateAllVersions`, they will already have run the same `applies_to?` checks. 

So when we call `UpdateAllVersions` from inside the `directories.each` loop, we already know the operation is correct for the job

Reply:
That's correct! CreateGroupUpdatePullRequest also returns false.

These classes are begging for a refactor, I am going to do that soon.